src/reporters/basic.ts

```diff
@@ -1,5 +1,5 @@
 import { consola } from '../utils/cli'
-import { Reporter } from 'src/types'
+import { Reporter } from '../types'
```

Comment:
Was this change necessary?

Reply:
Caused error in the editor:

![Screenshot 2023-11-17 at 11 40 45](https://github.com/unjs/webpackbar/assets/153197/6bdb5688-a115-48d2-8f24-dc106c0e05cd)

If you insist then I can look into fixing that (probably a missing mapping or `baseDir` in `tsconfig` but I though it's better to be consistent and use paths relative to the current file anyway.

Reply:
Also this change doesn't affect compiled bundle, as far as I can tell, so IMO keeping this change is better.

Reply:
Yeah it is better (hate old me that didn't used relative haha) only preferred that if it was not part of this commit but also okay 👍🏼